lib/enigma.rb

```diff
@@ -1,13 +1,57 @@
+# require './shift'
+
 class Enigma
-  attr_reader :date, :date_today
+  attr_reader :date, :date_today, :character_list
 
   def initialize
     @date = date
     @date_today = Time.now.strftime("%m/%d/%y").gsub("/", "").to_i
+    @shift_infomation = Shift.new
+    @character_list = ("a".."z").to_a << " "
+  end
+
+  def encrypt(message, key = nil, date = nil)
+    # require "pry"; binding.pry
+    {
+      encryption: msg_fuser(msg_converter(message, key, date)) ,
+      key: key,
+      date: date
+    }
   end
 
-  def encrypt(message, key, date)
-    require "pry"; binding.pry
+  def msg_converter(message, key = nil, date = nil)
```

Comment:
this does the heavy lifting for converting the messages. It takes in all arguments and returns a broken version of the encrypted method